code/modules/virus2/disease2.dm

```diff
@@ -16,11 +17,17 @@ var/global/list/disease2_list = list()
 	var/logged_virusfood=0
 
 /datum/disease2/disease/New(var/notes="No notes.")
-	uniqueID = rand(0,10000)
+	generate_uniqueID()
 	log += "<br />[timestamp()] CREATED - [notes]<br>"
-	disease2_list["[uniqueID]"] = src
 	..()
 
+/datum/disease2/disease/proc/generate_uniqueID()
+	for (var/x = 1 to 5)
+    	uniqueID = rand(...)
```

Comment:
easy to skim over code that looks correct
I did realize it almost immediately after of course https://github.com/d3athrow/vgstation13/pull/13141/commits/5beeb798a86f74419f2df66d8a92599f8a9627ab